spec/jobs/rate_limit_checker_job_spec.rb

```diff
@@ -1,34 +1,23 @@
 require "rails_helper"
```

Comment:
we can add `sidekiq/testing` to the `rails_helper` file and configure proper sidekiq testing: https://github.com/mperham/sidekiq/wiki/Testing